readme.md

```diff
@@ -28,18 +28,17 @@ $ npm install find-up
 ```
 /
 └── Users
-		└── sindresorhus
-				├── unicorn.png
-				└── foo
-						└── bar
-								├── baz
-								└── example.js
+	└── sindresorhus
```

Comment:
This is an improvement but I feel like space for indent would be better than tab.